VultisigApp/VultisigApp/View Models/KeysignViewModel.swift

```diff
@@ -152,9 +159,14 @@ class KeysignViewModel: ObservableObject {
         let keysignReq = TssKeysignRequest()
         keysignReq.localPartyKey = self.vault.localPartyID
         keysignReq.keysignCommitteeKeys = self.keysignCommittee.joined(separator: ",")
+        
         if let keysignPayload {
             keysignReq.derivePath = keysignPayload.coin.coinType.derivationPath()
+        } else {
+            // TODO: Should we use Ether as default derivationPath?
```

Comment:
This is only relevant to ECDSA , maybe we should ask user which blockchain they are signing tx for